core/tests/utils.py

```diff
@@ -5,7 +5,7 @@ def async_test(coroutine):
     """Run an async test to completion."""
 
     def _test(self):
-        asyncio.get_event_loop().run_until_complete(coroutine(self))
+        asyncio.new_event_loop().run_until_complete(coroutine(self))
```

Comment:
Been a while since I've used `aysncio` but could this be abstracted a bit more and just call `asyncio.run()`?

Reply:
You're right - we didn't historically use `run()` because it was only introduced in 3.7; but now that 3.7 is our minimum, we can use it.